gateway/src/apicast/policy/upstream/apicast-policy.json

```diff
@@ -5,7 +5,9 @@
   "description":
     ["This policy allows to modify the host of a request based on its path. ",
      "It accepts regular expressions that, when matched against the path, ",
-     "replace it with a given string."],
+     "replace it with a given string. \n",
+     "When combined with the Apicast policy, the upstream policy should be ",
```

Comment:
It should be spelled as APIcast policy:
https://github.com/3scale/apicast/blob/99d17ba34fa3ad7b461049b45e25a1088560d809/gateway/src/apicast/policy/apicast/apicast-policy.json#L3

Reply:
Right. Fixed.